src/util/importROAM.ts

```diff
@@ -0,0 +1,37 @@
+import { SimplePath, Timestamp } from '../types'
+import { State } from './initialState'
+import { importJSON } from './importJSON'
+import { Block } from '../action-creators/importText'
+import { RoamBlock, RoamPage } from 'roam'
+import { timestamp } from './timestamp'
+
+/**
+ * Recursively converts the roam children to blocks.
+ */
+const convertRoamBlocksToBlocks = (children: RoamBlock[]): Block[] => children.map(({ string, children, 'edit-time': editTime, 'create-time': createTime }) => ({
+  scope: string,
+  created: new Date(createTime).toISOString() as Timestamp,
+  lastUpdated: editTime ? new Date(editTime).toISOString() as Timestamp : timestamp(),
+  children: children && children.length ? convertRoamBlocksToBlocks(children) : []
+}))
+
+/**
+ * Converts the ROAM JSON to an array of blocks.
+ */
+const roamJsontoBlocks = (ROAM: RoamPage[]) => {
+
+  return ROAM.map((item: RoamPage) => {
+    return {
+      scope: item.title,
+      children: convertRoamBlocksToBlocks(item.children)
+    }
+  })
```

Comment:
I recommend the more succinct inline functions when the function body consists only of a return statement:

```js
  return ROAM.map((item: RoamPage) => ({
    scope: item.title,     
    children: convertRoamBlocksToBlocks(item.children)
  }))
```

Reply:
Yes, I observed this as well. Updated now 👍